src/current/v23.2/live-migration-service.md

```diff
@@ -95,6 +88,19 @@ To configure the LMS, override the [Helm chart values](https://github.com/cockro
 
 This section describes the most important and commonly used values. For details on all configurable values, refer to the [`values.yaml`](https://github.com/cockroachdb/molt-helm-charts/blob/main/lms/values.yaml) file.
 
+#### LMS version
+
+~~~ yaml
+image:
+  tag: 0.2.4  
+~~~
+
+`image.tag` specifies the LMS version. This **must** match the installed [`molt-lms-cli`](#molt-lms-cli) version.
```

Comment:
Might worth noting the `molt-lms-cli` version can be queried with `molt-lms-cli version`